firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -2090,7 +2090,11 @@ class BrowserViewController: UIViewController,
                 return
             }
             // TODO: FXIOS-10165 - Pass in the proper values based on top sites and other homepage links
-            navigationHandler?.navigateFromHomePanel(to: url, visitType: .link, isGoogleTopSite: false)
+            navigationHandler?.navigateFromHomePanel(
+                to: url,
+                visitType: .bookmark,
```

Comment:
need to confirm if we want this as `.link` or `.bookmark`

Reply:
confirmed that this should be `.link` even if we had it `.bookmark` previously. This impacts our frecency algorithm.